static/app/views/replays/detail/network/index.tsx

```diff
@@ -248,10 +255,17 @@ const StyledPanelTable = styled(PanelTable)<{columns: number}>`
   }
 `;
 
-const StatusPlaceHolder = styled(Placeholder)`
+const StatusContainer = styled('div')`
```

Comment:
I would name this `StatusBubble`

Reply:
Should we drop the bubble? 

Latest designs have dropped it:
<img width="656" alt="Screen Shot 2022-08-18 at 9 13 48 AM" src="https://user-images.githubusercontent.com/187460/185443820-d9f925d1-e623-4736-850e-d157ab4bc72a.png">